src/lib/components/00-atoms/Alert/index.js

```diff
@@ -30,41 +35,81 @@ const useStyles = makeStyles(theme => ({
     },
     "& .MuiAlert-icon": {
       paddingTop: "8px" // overrides default 7px to center icon better with margin 0 of text
+    },
+    "& .MuiAlert-root": {
+      display: "flex"
+    },
+    "& .MuiAlert-action": {
+      marginLeft: 0,
+      paddingLeft: "0.5em",
+      flexGrow: "1"
+    },
+    "& .MuiButtonBase-root": {
+      marginLeft: "auto !important"
+    },
+    "& .MuiLink-root": {
+      color: "green",
+      padding: "2px 0.5em 0 0"
```

Comment:
Pixel padding here. Not good.

Reply:
Related Material UI styles. 0.1em or such, might work or not.